watermarkchurch/theporch-apollos
# Add onboarding flag to app storage

#12 

This makes it so a user only sees Onboarding once.

To test:

- Open app (User should see Landing Screen)
- Go through onboarding
- Reload app (User should see Tabs screen)

A temporary Onboarding reset is in the connect tab. If you click the gear the next time you reload the app you should see the Landing Screen and onboarding flow. This will be removed before merge.
![image](https://user-images.githubusercontent.com/2528817/76128694-9d945d00-5fca-11ea-8b8a-378b37fb792d.png)
